### ISSUE TITLE
Blacklist option causes pipeline to fail

### ISSUE THREAD

**Issue by StevenWingett** [nf-core/atacseq#268]
### Description of the bug

If I specify a blacklist (a plain text BED file), the ATAC-seq pipeline fails after running for only a few seconds to a minute. 

The number of processes submitted varies each time I try the command, but the same error message is displayed each time:

    Invalid method invocation `pow` with arguments: 2 (java.lang.Integer), [:] (groovy.util.ConfigObject) on Math type.

If I don't specify a blacklist, the pipeline successfully runs to completion.

### Command used and terminal output

```console
[swingett@hal blacklist_problem]$ nextflow run nf-core/atacseq -r 2.0 --input samplesheet.csv --genome homo_sapiens.GRCh38.release_102 -config /public/singularity/containers/nextflow/lmb-nextflow/lmb.config --read_length 100 --aligner bowtie2 --deseq2_vst --outdir results --mito_name MT --blacklist ENCFF356LFX.bed.chr_deleted.txt -bg
[swingett@hal blacklist_problem]$ N E X T F L O W  ~  version 22.10.7
Launching `https://github.com/nf-core/atacseq` [big_einstein] DSL2 - revision: 0add18866a [2.0]


------------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/atacseq v2.0
------------------------------------------------------
Core Nextflow options
  revision                  : 2.0
  runName                   : big_einstein
  containerEngine           : singularity
  launchDir                 : /cephfs2/ngs/wingett/as_test/analysis/blacklist_problem
  workDir                   : /cephfs2/ngs/wingett/as_test/analysis/blacklist_problem/work
  projectDir                : /lmb/home/swingett/.nextflow/assets/nf-core/atacseq
  userName                  : swingett
  profile                   : standard
  configFiles               : /lmb/home/swingett/.nextflow/assets/nf-core/atacseq/nextflow.config, /public/singularity/containers/nextflow/lmb-nextflow/lmb.config

Input/output options
  input                     : samplesheet.csv
  read_length               : 100
  outdir                    : results

Reference genome options
  genome                    : homo_sapiens.GRCh38.release_102
  fasta                     : /public/genomics/species_references/nextflow/Genome_References/Ensembl/homo_sapiens/GRCh38/Release_102/FASTA/homo_sapiens__GRCh38__release102.dna.fa
  gtf                       : /public/genomics/species_references/nextflow/Genome_References/Ensembl/homo_sapiens/GRCh38/Release_102/GTF/Homo_sapiens.GRCh38.102.gtf
  bowtie2_index             : /public/genomics/species_references/nextflow/Genome_References/Ensembl/homo_sapiens/GRCh38/Release_102/Bowtie2_index/
  star_index                : /public/genomics/species_references/nextflow/Genome_References/Ensembl/homo_sapiens/GRCh38/Release_102/STAR_index/homo_sapiens.GRCh38.dna.102.STAR_index/
  blacklist                 : ENCFF356LFX.bed.chr_deleted.txt
  mito_name                 : MT

Alignment options
  aligner                   : bowtie2

Institutional config options
  config_profile_description: The MRC-LMB cluster profile provided by nf-core/configs.
  config_profile_contact    : Steven Wingett
  config_profile_url        : https://www2.mrc-lmb.cam.ac.uk/

Max job request options
  max_cpus                  : 112
  max_memory                : 700 GB
  max_time                  : 168.h

!! Only displaying parameters that differ from the pipeline defaults !!
------------------------------------------------------
If you use nf-core/atacseq for your analysis please cite:

* The pipeline
  https://doi.org/10.5281/zenodo.2634132

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x

* Software dependencies
  https://github.com/nf-core/atacseq/blob/master/CITATIONS.md
------------------------------------------------------
WARN: =============================================================================
  --macs_gsize parameter has not been provided.
  It will be auto-calculated by 'khmer unique-kmers.py' using the '--read_length' parameter.
  Explicitly provide '--macs_gsize macs2_genome_size' to change this behaviour.
===================================================================================
[81/8a6682] Submitted process > NFCORE_ATACSEQ:ATACSEQ:PREPARE_GENOME:CUSTOM_GETCHROMSIZES (homo_sapiens__GRCh38__release102.dna.fa)
[b4/bc4466] Submitted process > NFCORE_ATACSEQ:ATACSEQ:PREPARE_GENOME:GTF2BED (Homo_sapiens.GRCh38.102.gtf)
[24/53b1de] Submitted process > NFCORE_ATACSEQ:ATACSEQ:INPUT_CHECK:SAMPLESHEET_CHECK (samplesheet.csv)
[f9/57825a] Submitted process > NFCORE_ATACSEQ:ATACSEQ:PREPARE_GENOME:KHMER_UNIQUEKMERS (homo_sapiens__GRCh38__release102.dna.fa)
[69/1d4bd7] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (preB_aged_REP3_T1)
[f2/3c9bbe] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (preB_aged_REP4_T1)
[22/c354c3] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (preB_young_REP3_T1)
[c9/860871] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (preB_aged_REP1_T1)
[76/80ec96] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (preB_young_REP1_T1)
[5a/fa0002] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (preB_aged_REP2_T1)
Invalid method invocation `pow` with arguments: 2 (java.lang.Integer), [:] (groovy.util.ConfigObject) on Math type


-[nf-core/atacseq] Pipeline completed with errors-
WARN: Killing running tasks (8)
[76/fb61fc] Submitted process > NFCORE_ATACSEQ:ATACSEQ:FASTQ_FASTQC_UMITOOLS_TRIMGALORE:FASTQC (proB_aged_REP2_T1)
```

### Relevant files

[nf_core_atacseq_bug_files.zip](https://github.com/nf-core/atacseq/files/11139843/nf_core_atacseq_bug_files.zip)

### System information

nextflow version 22.10.7.5853
HPC
slurm
Singularity
AlmaLinux 8.7 (Stone Smilodon)
nf-core/atacseq  v2.0

**Comment by JoseEspinosa** [comment]
This should be fixed now in `dev` in case you want to give it a try

**Comment by JoseEspinosa** [comment]
Available from version 2.1.0

**Comment by StevenWingett** [comment]
Hi Jose,

Thank you very much for assisting me with this blacklist issue!!  I shall update my nf-core ATAC-seq pipeline shortly.

Many thanks,

Steven
